package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Portfolio site",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack --display-error-details && grunt"
```

Comment:
The flag `--display-error-details` is there so we actually can se whats going wrong, it it does.